src/Helper/RequestError.php

```diff
@@ -1,73 +1,69 @@
 <?php declare(strict_types=1);
-/**
- * If you want to add improvements, please create a fork in our GitHub:
- * https://github.com/myparcelnl
- *
- * @author      Reindert Vetter <reindert@myparcel.nl>
- * @copyright   2010-2020 MyParcel
- * @license     http://creativecommons.org/licenses/by-nc-nd/3.0/nl/deed.en_US  CC BY-NC-ND 3.0 NL
- * @link        https://github.com/myparcelnl/sdk
- * @since       File available since Release v2.0.0
- */
 
 namespace MyParcelNL\Sdk\src\Helper;
 
+/**
+ * @since v2.0.0
+ */
 class RequestError
 {
     /**
-     * @param $error
-     * @param array $result
+     * @param        $error
+     * @param  array $result
```

Comment:
Waar is die spatie voor, is dat een standaard zodat 'string' ook zou passen of is het een vergissing?